### DISCUSSION TITLE
spring boot project ， when add http 4.x plugin  , Multiple data sources can't Automatic switching

### DISCUSSION THREAD

**Discussion by leaf0603** [apache/skywalking#12307] [Q&A]
we use skywalking-java-agent : 9.2  , sky-walking server:9.7

error info : ERROR com.alibaba.druid.pool.DruidDataSource - {dataSource-2} init error

when delete http 4.x plugin ,  Function restored to normal

**Comment by CzyerChen** [comment]
It's better to give the specific plugin name and error log context. Your description is too brief to figure out.
